frontend/public/components/factory/list.tsx

```diff
@@ -402,10 +408,18 @@ export const List = connect(stateToProps, {sortList: UIActions.sortList})(
       selector: PropTypes.object,
       staticFilters: PropTypes.array,
       virtualize: PropTypes.bool,
+      currentSortField: PropTypes.string,
```

Comment:
Had to adjust the `PropTypes` to match the TypeScript props `type`, which is pretty neat that it checks it now.